docs/modules/ROOT/pages/tmail-backend/configure/index.adoc

```diff
@@ -215,3 +215,19 @@ Specified to TMail backend, we can configure the following configurations in the
 | https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys[S3 access key secret]
 |===
 
+== OpenPaas Integration Configurations
+
+Specified to TMail backend, we can configure the following configurations in the *openpaas.properties* file.
+
+.openpaas.properties additional content
+|===
+| Property name | explanation
+| rabbitmq.uri
+| URI for RabbitMQ server to communicate with OpenPaas. Example: `rabbitmq.uri=amqp://guest:guest@localhost:5672/`
+| openpaas.api.uri
+| URI for OpenPaas API. Example: `openpaas.api.uri=http://localhost:8080`
+| openpaas.admin.user
+| Admin username for OpenPaas. Example: `openpaas.admin.user=admin`
+| openpaas.admin.password
+| Admin password for OpenPaas. Example: `openpaas.admin.password=admin`
+|===
```

Comment:
What will it do?

Reply:
It's documented in openpaas-integration, I will add a link to there.